src/core/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -522,6 +522,9 @@
   <data name="RemoveButton.AutomationProperties.Name" xml:space="preserve">
     <value>Remove</value>
   </data>
+  <data name="RemoveTooltip.Text" xml:space="preserve">
+    <value>Remove</value>
```

Comment:
Can we add in comment "removes a user defined setting group for image resizer"

Add same line for 523.